content/en/docs/Writing policies/mutate.md

```diff
@@ -533,3 +533,5 @@ spec:
               name: "{{ element.name }}"           
               image: "{{ registry.io/{{images.containers.{{element.name}}.path}}:{{images.containers.{{element.name}}.tag}} }}"
 ```
+
+Note that the `patchStrategicMerge` is applied to the `request.object` (or the `request.oldObject` for `DELETE` requests). Hence, the patch needs to begin with `spec`.
```

Comment:
The mutate webhook does not forward DELETE requests, only CREATE and UPDATE.

Reply:
removed.